### PR TITLE
apt-get: add recommendation for apt

### PR DESCRIPTION
**Pull request by MIfoodie**
Copied from line 4 of [apt.md](https://github.com/tldr-pages/tldr/blob/main/pages/linux/apt.md) to recommend users to use `apt` instead of `apt-get` in Ubuntu versions 16.04 and later.

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**
Latest/All

### CODE REVIEW

File: pages/linux/apt-get.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 > Debian and Ubuntu package management utility.
 > Search for packages using `apt-cache`.
+> Recommended to use apt when used interactively in Ubuntu versions 16.04 and later.
```
Comment (fazlearefin) [inline pages/linux/apt-get.md:5]:
```suggestion
> Recommended to use `apt` when used interactively in Ubuntu versions 16.04 and later.
```

**Review by fazlearefin — CHANGES_REQUESTED** [review]

**Review by fazlearefin — APPROVED** [review]

File: pages/linux/apt-get.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 > Debian and Ubuntu package management utility.
 > Search for packages using `apt-cache`.
+> Recommended to use `apt` when used interactively in Ubuntu versions 16.04 and later.
```
Comment (gutjuri) [inline pages/linux/apt-get.md:5]:
```suggestion
> It is recommended to use `apt` when used interactively in Ubuntu versions 16.04 and later.
```

**Review by gutjuri — COMMENTED** [review]
Thanks for your contribution! I would rephrase the sentence slightly. Apart from that, LGTM
